src/System.Runtime.Serialization.Formatters/src/Resources/Strings.resx

```diff
@@ -129,4 +129,223 @@
+  </data>
+  <data name="Serialization_TypeExpected" xml:space="preserve">
+    <value>Invalid expected type.</value>
+  </data>
+  <data name="Serialization_StreamEnd" xml:space="preserve">
+    <value>End of Stream encountered before parsing was completed.</value>
+  </data>
+  <data name="Serialization_CrossAppDomainError" xml:space="preserve">
+    <value>Cross-AppDomain BinaryFormatter error; expected '{0}' but received '{1}'.</value>
+  </data>
+  <data name="Serialization_Map" xml:space="preserve">
+    <value>No map for object '{0}'.</value>
+  </data>
+  <data name="Serialization_Assembly" xml:space="preserve">
+    <value>No assembly information is available for object on the wire, '{0}'.</value>
+  </data>
+  <data name="Serialization_ObjectTypeEnum" xml:space="preserve">
+    <value>Invalid ObjectTypeEnum {0}.</value>
+  </data>
+  <data name="Serialization_AssemblyId" xml:space="preserve">
+    <value>No assembly ID for object type '{0}'.</value>
+  </data>
+  <data name="Serialization_ObjectTypeEnum" xml:space="preserve">
+    <value>Invalid ObjectTypeEnum {0}.</value>
+  </data>
+  <data name="Serialization_ArrayType" xml:space="preserve">
+    <value>Invalid array type '{0}'.</value>
+  </data>
+  <data name="Serialization_TypeCode" xml:space="preserve">
+    <value>Invalid type code in stream '{0}'.</value>
+  </data>
+  <data name="Serialization_TypeWrite" xml:space="preserve">
+    <value>Invalid write type request '{0}'.</value>
+  </data>
+  <data name="Serialization_TypeRead" xml:space="preserve">
+    <value>Invalid read type request '{0}'.</value>
+  </data>
+  <data name="Serialization_AssemblyNotFound" xml:space="preserve">
+    <value>Unable to find assembly '{0}'.</value>
+  </data>
+  <data name="Serialization_InvalidFormat" xml:space="preserve">
+    <value>The input stream is not a valid binary format. The starting contents (in bytes) are: {0} ...</value>
+  </data>
+  <data name="ArgumentNull_Stream" xml:space="preserve">
+    <value>Stream cannot be null.</value>
+  </data>
+  <data name="Serialization_TopObject" xml:space="preserve">
+    <value>No top object.</value>
+  </data>
+  <data name="Serialization_XMLElement" xml:space="preserve">
+    <value>Invalid element '{0}'.</value>
+  </data>
+  <data name="Serialization_TopObjectInstantiate" xml:space="preserve">
+    <value>Top object cannot be instantiated for element '{0}'.</value>
+  </data>
+  <data name="Serialization_ArrayTypeObject" xml:space="preserve">
+    <value>Array element type is Object, 'dt' attribute is null.</value>
+  </data>
+  <data name="Serialization_TypeMissing" xml:space="preserve">
+    <value>Type is missing for member of type Object '{0}'.</value>
+  </data>
+  <data name="ArgumentNull_Graph" xml:space="preserve">
+    <value>Object Graph cannot be null.</value>
```

Comment:
Nit: This one can also probably be removed. The one place it is used the param is already named "graph" so it should be clear what the issue is with just the default `ArgumentNullException` message.